File: LeetCode-October-Challenge/CheckiftheSentenceIsPangram.java
class Solution {
    public boolean checkIfPangram(String sentence) {
        int res = 0;
        for(Character x: sentence.toCharArray()){
            int ch = x - 'a';
            int bits = 1 << ch;
            res |= bits;
        }
        return res == (1 << 26) - 1;
    }
}
public class CheckiftheSentenceIsPangram {
    public static void main(String[] args) {
        String sentence = "thequickbrownfoxjumpsoverthelazydog";
        Solution s = new Solution();
        System.out.println(s.checkIfPangram(sentence));
    }
}

File: LeetCode-October-Challenge/CheckiftheSentenceIsPangram.js
var checkIfPangram = function(sentence) {
    let res = 0;
    for(let x of sentence){
        let ch = x.charCodeAt(0) - 97;
        let bits = 1 << ch;
        res |= bits;
    }
    return res === (1 << 26) -1;
};
let sentence = "thequickbrownfoxjumpsoverthelazydog";
console.log(checkIfPangram(sentence));